README.md

```diff
@@ -162,6 +162,9 @@ CloudGraph AWS Provider will ask you what regions you would like to crawl and wi
 | ses                         |                                                                                                                                                                                                                                                                                                                                                                               |
 | sns                         | kms, cloudtrail, cloudwatch, s3                                                                                                                                                                                                                                                                                                                                               |
 | sqs                         | elasticBeanstalkEnv, s3                                                                                                                                                                                                                                                                                                                                                       |
+| ssmActivation               |                                                                                                                                                                                                                                                                                                                                                                               |
+| ssmAssociation              |                                                                                                                                                                                                                                                                                                                                                                               |
+| ssmDocument                 |                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
mmm isnt ssm document the same as systemsManagerDocument?

Reply:
@tyler-dunkel oh.. didn't see that. I think If this is the case, I need to delete my ssmDocument and change the name `ssmActivation` and `ssmAssociation` to  `systemsManagerActivation` and `systemsManagerAssociation`
Also, I found a connection between `systemsManagerInstance` and `systemsManagerAssociation`. Will add this as well.